Document Download OTel log for Fusion job runs (#9471)
fishtown-analytics/docs.getdbt.com: website/docs/docs/deploy/run-visibility.md

</VersionBlock>

#### Downloading logs {#access-logs}

- To download logs for an individual step, select the step in the **Run summary** tab and click **Download** > **Download logs**. 
- Note that when viewing debug logs, the log output is truncated. To view and export all debug logs for an individual step, click **Download** > **Download all debug logs**.

<VersionBlock firstVersion="2.0">

> - For <Constant name="fusion_engine" /> runs on dbt command steps (such as **Invoke dbt with `dbt build`**), you can download the step's OpenTelemetry (OTel) telemetry as a Parquet file. Select the step and click **Download** > **Download OTel log**. This option appears only when the step emitted an OTel artifact. Some steps, such as `dbt deps`, don't produce one. For analysis examples, refer to [Querying telemetry data](/docs/fusion/telemetry-observability#querying-telemetry-data).

Author: ```suggestion
- You can download OpenTelemetry (OTel) logs for <Constant name="fusion" /> job command steps as a Parquet file. The file contains structured step-level log data that you can query or inspect outside dbt.

To download it, go to the **Run summary** tab in the job, select a step and click **Download** > **Download OTel log** to export a Parquet file

This option only appears when the step emitted an OTel log artifact.  Some steps, such as `dbt deps`, don't produce one. For analysis examples, refer to [Querying telemetry data](/docs/fusion/telemetry-observability#querying-telemetry-data).
```

> - For <Constant name="fusion_engine" /> runs on dbt command steps (such as **Invoke dbt with `dbt build`**), you can download the step's OpenTelemetry (OTel) telemetry as a Parquet file. Select the step and click **Download** > **Download OTel log**. This option appears only when the step emitted an OTel artifact — some invoke steps, such as `dbt deps`, do not produce one. For analysis examples, refer to [Querying telemetry data](/docs/fusion/telemetry-observability#querying-telemetry-data).

Author: ```suggestion
- For <Constant name="fusion_engine" /> runs on dbt command steps (such as **Invoke dbt with `dbt build`**), you can download the step's OpenTelemetry (OTel) telemetry as a Parquet file. Select the step and click **Download** > **Download OTel log**. This option appears only when the step emitted an OTel artifact. Some steps, such as `dbt deps`, don't produce one. For analysis examples, refer to [Querying telemetry data](/docs/fusion/telemetry-observability#querying-telemetry-data).
```

</VersionBlock>

#### Downloading logs {#access-logs}

- To download logs for an individual step, select the step in the **Run summary** tab and click **Download** > **Download logs**. 
- Note that when viewing debug logs, the log output is truncated. To view and export all debug logs for an individual step, click **Download** > **Download all debug logs**.

- You can download OpenTelemetry (OTel) logs for <Constant name="fusion" /> job command steps as a Parquet file. The file contains structured step-level log data that you can query or inspect outside dbt.

To download it, go to the **Run summary** tab in the job, select a step and click **Download** > **Download OTel log** to export a Parquet file

> This option only appears when the step emitted an OTel log artifact.  Some steps, such as `dbt deps`, don't produce one. For analysis examples, refer to [Querying telemetry data](/docs/fusion/telemetry-observability#querying-telemetry-data).

Author: ```suggestion
This option only appears when the step emitted an OTel log artifact.  Some steps, such as `dbt deps`, don't produce one. For analysis examples, refer to [Querying telemetry data](/docs/fusion/telemetry?version=2.0&name=Fusion#querying-telemetry-data).
```

> <Lightbox src="/img/docs/dbt-platform/deployment/download-logs.png" width="85%" title="Download logs" />

Reviewer: Can we update this screenshot to include OTEL logs

<img width="1927" height="1023" alt="Image" src="https://github.com/user-attachments/assets/3163a51a-5a1d-4402-85aa-8117b9dfdaa0" />

Author: good call!

</VersionBlock>

#### Downloading logs {#access-logs}

- To download logs for an individual step, select the step in the **Run summary** tab and click **Download** > **Download logs**. 
- Note that when viewing debug logs, the log output is truncated. To view and export all debug logs for an individual step, click **Download** > **Download all debug logs**.

> <VersionBlock firstVersion="2.0">

Author: ```suggestion
```

- You can download OpenTelemetry (OTel) logs for <Constant name="fusion" /> job command steps as a Parquet file. The file contains structured step-level log data that you can query or inspect outside dbt.

To download it, go to the **Run summary** tab in the job, select a step and click **Download** > **Download OTel log** to export a Parquet file

This option only appears when the step emitted an OTel log artifact.  Some steps, such as `dbt deps`, don't produce one. For analysis examples, refer to [Querying telemetry data](/docs/fusion/telemetry-observability#querying-telemetry-data).

> </VersionBlock>

Author: ```suggestion
```
